lib/disability_compensation/providers/document_upload/lighthouse_supplemental_document_upload_provider.rb

```diff
@@ -68,4 +75,46 @@ def log_upload_failure(uploading_class_prefix, error_class, error_message)
       }
     )
   end
+
+  private
+
+  def handle_lighthouse_response(api_response)
+    response_body = api_response.body['data']
+
+    if response_body['success'] == true && response_body['requestId']
+      Lighthouse526DocumentUpload.create!(
+        form526_submission_id: @submission_id,
+        # document_type: Lighthouse526DocumentUpload::BDD_INSTRUCTIONS_DOCUMENT_TYPE,
+        document_type: polling_record_document_type,
+        lighthouse_document_request_id: response_body['requestId']
+      )
+
+      StatsD.increment("#{@statsd_metric_prefix}.#{STATSD_PROVIDER_METRIC}.#{STATSD_SUCCESS_METRIC}")
+    end
+  end
+
+  def polling_record_document_type
+    case @uploading_class
+    when UploadBddInstructions
+      Lighthouse526DocumentUpload::BDD_INSTRUCTIONS_DOCUMENT_TYPE
+    end
+  end
+
+  # Creates a Lighthouse526DocumentUpload record, where we save
+  # a unique 'request ID' Lighthouse provides us in the API response after we upload a document.
+  # We use this ID in the Form526DocumentUploadPollingJob chron job to check the status of the document
+  # after Lighthouse has received it.
+  #
+  # @param api_response [Faraday::Response] the response from the Lighthouse Benefits Documents API upload endpoint
+  def create_lighthouse_polling_record(api_response)
+    response_body = api_response.body['data']
+
+    if response_body['success'] == true && response_body['requestId']
+      Lighthouse526DocumentUpload.create!(
```

Comment:
it might be good to add logging in here too like `polling record created` or something, with the form526_submission_id, doc type, and lh doc req id all logged